### PR TITLE
Replace Bintray with Lightbend Registries

### PR DESCRIPTION
**Pull request by marcospereira**

### CODE REVIEW

**Review by marcoderama — APPROVED** [review]
Approved with ignorable comments

File: enterprise-suite/values.yaml [hunk 1]
```diff
@@ -57,7 +57,7 @@ configMapReloadImage: jimmidyson/configmap-reload
 # kube-state-metrics image
 kubeStateMetricsImage: gcr.io/google_containers/kube-state-metrics
 # go-dnsmasq image
-goDnsmasqImage: lightbend-docker-registry.bintray.io/lightbend/go-dnsmasq
+goDnsmasqImage:  docker.cloudsmith.io/lightbend/registry/lightbend-go-dnsmasq
```
[the remark this answers is not in the document]
Comment (marcospereira) [reply enterprise-suite/values.yaml]: It was the wrong registry too! 💣

**Comment by marcospereira** [comment]
@sebastian-alfers @JustinPihony I'm merging this myself so that I can test the release workflow on CircleCI.

File: enterprise-suite/gotests/resources/alert-app.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ spec:
       containers:
         # source: https://github.com/lightbend/k8s-explore/tree/master/query/nan
```
[the remark this answers is not in the document]
Comment (marcoderama) [reply enterprise-suite/gotests/resources/alert-app.yaml:18]: Sorry, "wrong" was wrong.  I just don't think this belongs here.  I'm not sure how it ended up there in the first place.  (I went hunting in the history and could find no _reason_ for it.)

Just in tests I realize, so not a big deal.

**Pull request merged**